dc/council/code/titles/24/sections/24-1102.xml

```diff
@@ -3,13 +3,14 @@
   <num>24-1102</num>
   <heading>Authority to enter into and execute Compact.</heading>
   <text>The Mayor is authorized to enter into and execute on behalf of the District a compact with any state or states legally joining therein in the form substantially as follows:</text>
+  <text>
+    <center>
+        <strong>THE INTERSTATE COMPACT FOR JUVENILES</strong>
+    </center>
+  </text>
   <para>
```

Comment:
Is it correct for this to be a `<para>`? All of this language isn't the language of the law per se. The operational portion of the law ends with the colon on line 5. The rest of this is more like a form agreement, isn't it? And for that type of text, is it appropriate to provide all this information. Put it another way, why not just have all of these be a series of `<text>`?